docs/application/dotnet/index.md

```diff
@@ -4,90 +4,20 @@ Tizen .NET is an exciting new way to develop applications for the Tizen operatin
 
 The existing Tizen frameworks are either C-based with no advantages of a managed runtime, or HTML5-based with fewer features and lower performance than the C-based solution. With Tizen .NET, you can use the C# programming language and the Common Language Infrastructure standards, and benefit from a managed runtime for faster application development and code execution that is efficient and secure.
 
-To start developing Tizen applications using .NET, see [Creating Your First Tizen .NET Application with Visual Studio](getting-started/first-app.md).
+The introduction to .Net applications documentation provides overall information :
 
-<a id="runtime"></a>
-## Managed Runtime Advantages
+- [Creating Your First Tizen .NET Application with Visual Studio](getting-started/first-app.md)
+  Demonstrates how you can create applications with Visual Studio.
```

Comment:
[Creating your first Tizen .NET application with Visual Studio](getting-started/first-app.md) explains how to create an application using Visual Studio.

Reply:
done